app/ports/templates/ports/about.html

```diff
@@ -0,0 +1,73 @@
+                </li>
+                <li>Makes available all the collected data through API.</li>
+            </ol>
+            <p>The application is based on the <a href="https://www.djangoproject.com">Django Framework</a> and runs
+                inside a <a href="https://www.docker.com">Docker</a> container with <a href="https://www.nginx.com">nginx</a>
+                and <a href="https://uwsgi-docs.readthedocs.io">uWSGI</a> serving the content. It has been written as a part of the <a href="https://summerofcode.withgoogle.com">
+                Google Summer of Code 2019</a> mentored by the <a href="https://www.macports.org">MacPorts</a> organisation.</p>
+        </div>
+    </div>
+    <br>
+    <h4>Credits</h4>
+    <hr>
+    <p>The MacPorts Webapp was written by <a href="https://github.com/arjunsalyan">Arjun Salyan</a> when MacPorts
+        participated in the Google Summer of Code 2019. The project was mentored by
+        <a href="https://github.com/mojca">Mojca Miklavec</a> and <a href="https://github.com/umeshksingla">Umesh Singla</a>.
+        The server is administered by <a href="https://github.com/neverpanic">Clemens Lang</a>.
+    </p>
+
+    <h5>Old Statistics Website</h5>
+    <p>
+	The old MacPorts statistics web application and the initial client were written by <a href="https://trac.macports.org/wiki/derek">Derek Ingrouville</a>
+	when MacPorts participated in <a href="https://trac.macports.org/wiki/SummerOfCode2011">Google Summer of Code 2011</a>. Some documentation for the
+	source written in GSoC can be found in <a href="https://trac.macports.org/wiki/MacPortsStatisticsGSoC2011">the Wiki</a>. The source code for the
+	server is available in MacPorts' Subversion repository at <a href="https://trac.macports.org/browser/branches/gsoc11-statistics">/branches/gsoc11-statistics</a>.
+    </p>
+    <br>
+    <h4>Dependencies</h4>
+    <hr>
+    The app is dependent on the following:
+    <ul>
+        <li><a href="https://build.macports.org/json/help">Buildbot JSON API</a>: Allows fetching the build history.</li>
+        <li><a href="https://github.com/macports/macports-contrib/blob/master/portindex2json/portindex2json.tcl">portindex2json.tcl</a>: Converts PortIndex into JSON format. Written by <a href="https://github.com/jmroot">Joshua Root.</a></li>
+        <li><a href="https://github.com/macports/macports-infrastructure/blob/master/jobs/mprsyncup">mprsyncup job</a>: Runs portindex2json.tcl and distributes PortIndex.json through rsync.</li>
+        <li><a href="https://github.com/macports/macports-ports/tree/master/sysutils/mpstats">mpstats</a>: Port that submits
+        installation statistics.
+        <br>
+            <p>
+	The client implementation as the <code>mpstats</code> port is the work of <a href="https://trac.macports.org/wiki/derek">Derek Ingrouville</a>,
+	<a href="https://trac.macports.org/wiki/jmr">Joshua Root</a> and <a href="https://trac.macports.org/wiki/cal">Clemens Lang</a>. The source of
```

Comment:
We need to decide whether to cite developers under their trac pages or under their github handles (or both at all locations). Maybe not do a mixture of both (one citation with one and the other with another) on the same page. I don't have a strong preference for either.

Reply:
I could not find the right github handle for Derek, rest I have changed.